### PR TITLE
Maggie bio profile update

### PR DESCRIPTION
**Pull request by Megmugure**

### CODE REVIEW

**Review by bkmgit — COMMENTED** [review]
Is it possible to upload the images to this repository directly? This makes managing the site easier since all content is self contained.

File: pages/en/committee.markdown [hunk 1]
```diff
@@ -7,19 +7,19 @@ toc_sticky: true
 lang: en
 ---
 ## Caleb Kibet (Chair)
-![Add Profile Photo](https://www.h3abionet.org/images/2019/nodes/45_photo_.jpg)
+![Caleb's Profile Photo](https://www.h3abionet.org/images/2019/nodes/45_photo_.jpg)
```
Comment (bkmgit) [inline pages/en/committee.markdown:10]:
```suggestion
![Caleb's Profile Photo](https://www.h3abionet.org/images/2019/nodes/45_photo_.jpg){: height="150" width="150" }
```

File: pages/en/committee.markdown [hunk 2]
```diff
@@ -7,19 +7,19 @@ toc_sticky: true
...
 
 Caleb Kibet is a bioinformatics researcher, a lecturer, an open science advocate, and a mentor. Caleb is currently a bioinformatician at _icipe_ and teaches bioinformatics at Pwani University. His research interests are in regulatory genomics, where he uses machine learning and statistical modelling to understand and predict transcription factor binding sites and their link to diseases. Caleb is also a founder of OpenScienceKE, an initiative that promotes open approaches to bioinformatics research in Kenya. Caleb is passionate about open science and reproducible bioinformatics research. He continually seeks out opportunities to spread open science, especially within the bioinformatics community in Kenya
   
 ## Pauline Karega
-![Add Profile Photo](https://bhki.org/wp-content/uploads/2021/01/1592590261534.jpeg)
+![Pauline's Profile Photo](https://bhki.org/wp-content/uploads/2021/01/1592590261534.jpeg)
```
Comment (bkmgit) [inline pages/en/committee.markdown:15]:
```suggestion
![Pauline's Profile Photo](https://bhki.org/wp-content/uploads/2021/01/1592590261534.jpeg){: height="150" width="150" }
```

File: pages/en/committee.markdown [hunk 3]
```diff
@@ -7,19 +7,19 @@ toc_sticky: true
...
 
 Pauline Karega is a bioinformatics student and researcher working on evolutionary bioilogy. She is a co-founder of the Bioinformatics Hub of Kenya initiative (BHKi) and enjoys working at a project management capacity. She is passionate about capacity building in bioinformatics and data science, and community outreach. She is a certified Carpentries instructor and is actively involved in carpentries training. Outside science she enjoys performing arts.    
   
 ## Margaret Wanjiku
-![Add Profile Photo](https://sbcf.fr/wp-content/uploads/2018/03/sbcf-default-avatar.png)
+![Margaret's Profile Photo](https://bhki.org/wp-content/uploads/2021/01/4ry06f-1-150x150.jpg)
```
Comment (bkmgit) [inline pages/en/committee.markdown:20]:
```suggestion
![Margaret's Profile Photo](https://bhki.org/wp-content/uploads/2021/01/4ry06f-1-150x150.jpg){: height="150" width="150" }
```

**Review by bkmgit — COMMENTED** [review]
Add picture sizing information so all pictures on site have similar size.

**Comment by bkmgit** [comment]
@Megmugure Thanks for the additions. Is it possible to upload the pictures to https://github.com/Megmugure/BOSS22/tree/Maggie_bio_profile/assets/images ?

**Comment by Megmugure** [comment]
> @Megmugure Thanks for the additions. Is it possible to upload the pictures to https://github.com/Megmugure/BOSS22/tree/Maggie_bio_profile/assets/images ?

Sure.

**Comment by Megmugure** [comment]
> Is it possible to upload the images to this repository directly? This makes managing the site easier since all content is self contained.

Done!

**Comment by bkmgit** [comment]
Looks good.

**Pull request merged**